react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -123,3 +123,51 @@ const enableLassoArgs = {
         },
     ],
 };
+
+export const lassoSelectionWithCallback: ComponentStory<
+    typeof DeckGLMap
+> = () => {
+    const [data, setData] = React.useState<string[]>([]);
+    const getSelectedWellsDataCallBack = React.useCallback(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (pickingInfos: any[]) => {
+            const selectedWells = pickingInfos
+                .map((item) => item.object)
+                .filter((item) => item.type === "Feature")
+                .map((item) => item.properties["name"]) as string[];
+            setData(selectedWells);
+        },
+        []
+    );
+    const lassoArgsWithSelectedWellsDataCallback: Record<string, unknown> = {
+        ...disableLassoArgs,
+        layers: [
+            {
+                "@@type": "WellsLayer",
+                data: "@@#resources.wellsData",
+            },
+            {
+                "@@type": "LassoLayer",
+                visible: true,
+                data: "@@#resources.wellsData",
```

Comment:
I hadn't noticed this before; is it necessary for the selection layer to have a copy of the data?

Reply:
I think now we can delete it for now at least.